package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@multiversx/sdk-core",
-  "version": "13.2.2",
+  "version": "13.2.3",
```

Comment:
Let's bump the minor version, so that developers take this update intentionally (not as a hotfix) - `v13.3.0`.